chevah_build

```diff
@@ -64,6 +65,7 @@ EXTRA_LIBRARIES_NO_CFFI="\
     python-modules/pycryptodome-${PYCRYPTODOME_VERSION} \
     python-modules/python-setproctitle-1.1.8.dev0 \
     python-modules/python-scandir-${SCANDIR_VERSION} \
+    python-modules/python-safety-${SAFETY_VERSION} \
```

Comment:
Do we need it from python-modules?
I think that we can get it via pip  from pypi.chevah.com

in python-modules we only used to have the code this is CPU/OS dependent

safety is an universal wheel..


Note that I have already start using safety for chevah/server see https://github.com/chevah/server/blob/master/pavement.py#L1861

Reply:
That's what I tried first, but it didn't work for all OS'es. I'll look closer when I'll come back to this PR.

Reply:
OK, so I took a closer look… The way things currently work in our build system, for platforms not built around cffi, `PIP_LIBRARIES` is uninitialized and these modules (PyOpenSSL, PyCryptodome etc.) are collected in `EXTRA_LIBRARIES` and built separately through `initialize_python_module()` using `setup.py`. Currently supported platforms in this situation: Solaris 10 and HP-UX.

What we can do here, I guess, is to get through `pip` the modules that would work, like `scandir` and `safety`, using something like `PIP_LIBRARIES_NO_CFFI` for uniformity. I've tried something like that for Solaris 10 and it worked: https://chevah.com/buildbot/builders/python-package-solaris-10/builds/72

Do you think this is worth implementing? The end result is identical, as far as I can tell.

Reply:
we need to create 2 types of  Python package/modules, similar to what we have for chevha/server.

Runtime:
* scandir
* cryptography
* pyopenssl

Build time / development helpers:
* brink
* safety
* ... etc

------

I don't want to have `safety` part of the python-package distribution.

Since `safety` has no CPU/OS depended part, we can install it separately in each project and be able to update `safety` without updating python-package.

So safety should be part of the pavement.py development script or implemented in a similar way

Reply:
Yes, it's better to not bundle `safety`. But I've already implemented `PIP_LIBRARIES_NO_CFFI` for Solaris 10 and HP-UX, hope you don't mind.

So `safety` is installed now only during the test phase, when actually needed. I'm not sure about `pavement.py`, that is not used when building or testing locally. What do you have in mind?